internal/store/pod.go

```diff
@@ -37,7 +37,7 @@ import (
 
 var (
 	descPodLabelsDefaultLabels = []string{"namespace", "pod", "uid"}
-	podStatusReasons           = []string{"Evicted", "NodeAffinity", "NodeLost", "Shutdown", "UnexpectedAdmissionError"}
+	podStatusReasons           = []string{"Evicted", "ImagePullBackOff", "NodeAffinity", "NodeLost", "Shutdown", "UnexpectedAdmissionError"}
```

Comment:
If I am not mistaken `ImagePullBackOff` is a container status not a pod status

Reply:
Oh, yes.